### ISSUE TITLE
What output should I expect during training?

### ISSUE THREAD

**Issue by timnugent** [zihangdai/xlnet#169]
I'm trying some fine-tuning using the base model. The last output to stdout by run_classifier.py is:

W0717 12:04:27.171862 140035128494976 deprecation.py:323] From /usr/local/lib/python3.6/dist-packages/tensorflow/python/training/saver.py:1066: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
I0717 12:04:29.599863 140035128494976 session_manager.py:500] Running local_init_op.
I0717 12:04:30.265308 140035128494976 session_manager.py:502] Done running local_init_op.
I0717 12:04:50.398686 140035128494976 basic_session_run_hooks.py:606] Saving checkpoints for 0 into esg_output/model/model.ckpt.
2019-07-17 12:05:32.302063: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0

.. but there has been nothing else for 4 hours. I can see that my two GPUs have their RAM maxed out, although the fans and power draw suggest not a lot is going on. A file was written to the output directory recently (events.out.tfevents.1562943806.labs-twin-gpu-1). Is this expected behaviour during training, or has it hung?

Thanks
Tim

**Comment by timnugent** [comment]
Scrap that - it wasn't using my GPUs properly so doing everything on the CPU and taking a very long time. Fixed the GPU issue and it's now outputting the loss etc

**Comment by piyush1146115** [comment]
> Scrap that - it wasn't using my GPUs properly so doing everything on the CPU and taking a very long time. Fixed the GPU issue and it's now outputting the loss etc

Would you please specify the process of fixing this issue? I am facing the same problem.

**Comment by timnugent** [comment]
Sorry - I can't recall the exact detail. From memory it was a combination of re-installing CUDA drivers (currently have 10.0 running ok on that machine) and removing tensorflow-gpu, then re-installing, reboot etc

**Comment by timnugent** [comment]
I think that when I ran nvidia-smi, it didn't show any cards (there were 2 in the machine). Driver re-install  + reboot and they showed up again with that command, then xlnet picked them up too

**Comment by piyush1146115** [comment]
> I think that when I ran nvidia-smi, it didn't show any cards (there were 2 in the machine). Driver re-install + reboot and they showed up again with that command, then xlnet picked them up too

Thanks a lot. It might be helpful for me.
